libr/core/cbin.c

```diff
@@ -1117,6 +1126,16 @@ static int bin_entry(RCore *r, int mode, ut64 laddr, int va, bool inifin) {
 				snprintf (str, R_FLAG_NAME_SIZE, "entry%i", i);
 			}
 			r_flag_set (r->flags, str, at, 1);
+			switch (entry->type) {
+			case R_BIN_ENTRY_TYPE_INIT:
+			case R_BIN_ENTRY_TYPE_FINI:
+			case R_BIN_ENTRY_TYPE_PREINIT:
+				if (haddr != UT64_MAX && hvaddr != UT64_MAX && (bits == 32 || bits == 64)) {
```

Comment:
one last thing. Did you consider using `entry->bits`?

Reply:
ee61b564011cacd96948b91fe5367609e130ed89 yeah that does work too (with less code needed) ... thanks!